blips/blip-2-researchcertificate.md

```diff
@@ -0,0 +1,112 @@
+# Research Object Certification
+
+| BLIP:     | 2                                                         |
+| -------- | ------------------------------------------------------------ |
+| Title:   | Research Object Certification                                      |
+| Owner(s):  | James Lawton                                                           |
+| Author(s):  | James Lawton                                                           |
+| Status:  | ![Draft](http://rfc.unprotocols.org/spec:2/COSS/draft.svg) |
+| Created: | 2020-02-18                                                   |
+| License: | BSD-2-Clause                                                 |
+
+## Motivation
+Certifying that specific data or documents are generated, modified, analyzed, or processed at a specific time is valuable for the scientific community in order to protect intellectual property and incentivize data sharing. A common methodology for this is trusted timestamping which is used to ensure the creation and modification time of arbitrary data or a document. This requires a trusted authority to validate the data integrity by digitally signing upon receipt of a hash derived from the data.
+
+In addition, as the research process can be considered a workflow - there is a large benefit to include measurements of how data has changed through the research process, for instance between initial acquisition and then preprocessing. Thus, by concatenating multiple generated certificates, the workflow of a research process can be certified and validated at a later point in time.
+
+Our aim is to standardize the research data certification with respect to what metadata should be included to identify the data, the process of research certificate generation, and the subsequent validation of an issued research certificate.
+
+## Specification
+Each certified piece of data is minted as a non-fungible (*Transferable ERC721 compliant or non-transferable) token. Each contract MUST include the ERC721Metadata standard augmented with an additional field that contains the hash of the data object. This is in order to ensure that the dataURI hosted offchain can be resolved to an onchain transaction. The ERC721Metadata is necessary to include the information for future identification of the data object.
+
+```solidity
+/// @title Research Object Metadata Extension 
+/// @dev See https://blips.bloxberg.org/blips/
+///  Note: the ERC-165 identifier for this interface is 0x5b5e139f.
+interface objectMetadata /* is ERC721 */ {
```

Comment:
```suggestion
interface objectMetadata is ERC721Metadata  {
```

New Solidity should allow normal inheritance in interfaces. Directly inherit from `ERC721Metadata`?
It is also nice to use suggestions like this.